src/overlays/actors/ovl_Tools/ac_tools.c

```diff
@@ -17,24 +18,143 @@ ActorProfile Tools_Profile = {
     /* */ sizeof(Tools),
     /* */ aTOL_actor_ct,
     /* */ aTOL_actor_dt,
-    /* */ (ActorFunc)none_proc1,
-    /* */ (ActorFunc)none_proc1,
+    /* */ (void*)none_proc1,
```

Comment:
In the gc repo, we do have it as a define with the cast already to avoid ugliness but I don't know if it's worth having it here lol